ci/operator.yaml

```diff
@@ -0,0 +1,3028 @@
+apiVersion: v1
```

Comment:
Please do not copy the operator.yaml here ...
please link to https://github.com/kiegroup/kogito-serverless-operator/blob/main/operator.yaml directly and remove this file.
We don't want to have to manage 2 times the same file in different places

Reply:
@radtriste isnt' the same this is the line needed to deploy the image in the registry https://github.com/kiegroup/kogito-serverless-operator/pull/93/files#diff-0f6092c791b7d9467439876ba331d3043b4f3f6d66843fa766c4a81ebce4df9aR2997

Reply:
I see you use it here: https://github.com/kiegroup/kogito-serverless-operator/pull/93/files#diff-14c4d45418ce873b2d2ec357d9d66b7e813d4ff11de6bec43202421143b37027R55
Then please bring on a bash command there to take the file https://github.com/kiegroup/kogito-serverless-operator/blob/main/operator.yaml, update the image and apply the modified content

Could be something like:
```
curl https://raw.githubusercontent.com/kiegroup/kogito-serverless-operator/main/operator.yaml | sed -E 's|image: quay.io/kiegroup/.*|image: image-registry.openshift-image-registry.svc:5000/kogito-serverless-operator-system/kogito-serverless-operator:latest|g' | kubectl apply -f -
```